tests/core/test_state_sync.py

```diff
@@ -211,12 +212,21 @@ def test_snapshots_exists(state_sync: EngineAdapterStateSync, snapshots: t.List[
     assert state_sync.snapshots_exist(snapshot_ids) == snapshot_ids
 
 
-def get_snapshot_intervals(state_sync, snapshot):
-    intervals = state_sync._get_snapshot_intervals([snapshot])[-1]
-    return intervals[0] if intervals else None
```

Comment:
This is a truly private method that was wrapped by a function for testing. By making this a fixture it can now be overriden.